cypress/e2e/CMS/$Biography/index.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="Cypress" />
+import { should } from 'chai';
+import { And, Then, Given, When } from 'cypress-cucumber-preprocessor/steps';
+
+
+When('user selects test {string}',(title) =>{
+    cy.get("input[value='Select Site Section']").should('be.visible').click({ force: true });
+
+}) 
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+    
+And('user clicks on {string} to a title section',(buttonLable) =>{
+    cy.get('#edit-field-title-add-more').contains(buttonLable).click({force:true})
+})
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
```

Comment:
1.this step is implemented in common, remove it here
2. this is a duplicate step from above